liesel/goose/builder.py

```diff
@@ -124,6 +134,14 @@ def set_initial_values(self, model_state: ModelState, multiple_chains=False):
 
         self._model_state = Option(model_states)
 
+    def set_init_startegies(self, init_strategies: InitStrategies):
+        self._init_strategies = init_strategies
+
+    @property
+    def init_startegies(self) -> InitStrategies:
```

Comment:
```py
def init_startegies(self) -> Option[InitStrategies]:
```

Probably, we should rather return `self._init_strategies.value` here to avoid user-exposed options, but let's keep it as is for now for consistency with the other properties.